docs/static/pipeline-viewer.asciidoc

```diff
@@ -0,0 +1,107 @@
+[role="xpack"]
+[[logstash-pipeline-viewer]]
+=== Pipeline Viewer UI
+
+NOTE: The pipeline viewer is an {xpack} feature that requires a
+paid {xpack} license. See the
+https://www.elastic.co/subscriptions[Elastic Subscriptions] page for
+information about obtaining a license.
+
+//REVIEWERS: Not sure what license is required here. Does this come with basic?
+
+The pipeline viewer in {xpack} provides a simple way for you to visualize and
+monitor the behavior of complex Logstash pipeline configurations. Within the
+pipeline viewer, you can explore a directed acyclic graph (DAG) representation
+of the overall pipeline topology, data flow, and branching logic. The diagram
+is overlayed with important metrics, like events per second and time spent in
+milliseconds, for each plugin in the view.
+
+The diagram includes visual indicators to draw your attention to potential
+bottlenecks in the pipeline, making it easy for you to diagnose and fix
+problems.
+
+Before using the pipeline viewer, you need to
+{logstash-ref}/setup-xpack.html[set up {xpack}] and configure
+{xpack-ref}/monitoring-logstash.html[Logstash monitoring]. 
+
+//REVIEWERS: Is anything else required to use the pipeline viewer?
```

Comment:
There should be a big note/info/reccomendation about using semantic IDs in configs. If the ID is not present in the configuration, we generate an automatic ID but this is not ideal for debugging purposes. This will be evident when there are 3-4 grok debuggers and its better if user knows which grok filter does what. So adding a semantic ID like `apacheParsingGrok` or `cloudwatchGrok` helps to narrow which of the many groks are slow. See https://github.com/elastic/x-pack-kibana/pull/2218